core/src/main/kotlin/br/com/concrete/yosef/api/DynamicViewCreator.kt

```diff
@@ -57,14 +60,45 @@ class DynamicViewCreator(
                     "not a ViewGroup.")
             }
 
-            it.forEach { addChildrenRecursively(parentView, it, listener) }
+            if (parentView is ListView) {
+                addChildrenToParentListView(it, parentView, context, listener)
+            } else {
+                it.forEach {
+                    addChildrenRecursively(parentView, it, listener)
+                }
+            }
         }
 
         component.applyProperties(parentView, componentSpec.dynamicProperties, listener)
 
         return parentView
     }
 
+    private fun addChildrenToParentListView(children: List<DynamicComponent>, listView: ListView, context: Context, listener: OnActionListener?) {
+        val listOfViews: ArrayList<View> = ArrayList()
+        children.forEach {
+            val comp = getComponentByType(it)
+            val view = comp.createView(listView.context)
+            it.children?.forEach { addChildrenRecursively(view as ViewGroup, it) }
+
+            val frameLayout = FrameLayout(context)
```

Comment:
why?

Reply:
We still need this because the when the view is added directly on the ListView we cant set margin properties because the LayoutParams is different, adding a FrameLayout before was the best way that i found to workaround this.